### PR TITLE
Adding Week 15 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding Weekly post

msedge_Vsq0IFuU9A

### CODE REVIEW

**Comment by skoudoro** [comment]
@xtanion ? @m-agour ?

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+--------------------------
+This week I started with highlighting the shapes. As discussed earlier I had two ways but while implementing them, I found out both ways aren't that good to continue with.
+The first way in which we thought of creating the scaled shapes in the background had an issue with the stacking. The border(blue rectangle) and the shape(grey rectangle) both seem to look like different shapes just grouped together as shown below.
+
+.. image:: https://user-images.githubusercontent.com/64432063/192321622-964cef6e-f965-4a24-8dcf-0b899fe5e387.gif
+    :width: 400
+    :align: center
+
+While playing around with the second way, which was to add yellow color to the shape to make it brighter, it was difficult to identify which shape was selected also sometimes instead of making it brighter the addition of color created a new color which again confused the user.
+After discussing these issues my mentors suggested having a user-defined highlight color that will be shown whenever the shape is selected.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:21]: Missing a comma after `After discussing these issues`

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.. image:: https://user-images.githubusercontent.com/64432063/192326416-4454718d-1dda-4a13-9f97-07387a50a580.gif
+    :width: 400
+    :align: center
+
+Along this, we were also trying to integrate shaders to the Actor2D (ie. the UI elements) but there were some issues with it. I used `this <https://fury.gl/latest/auto_tutorials/03_shaders/viz_shader.html#sphx-glr-auto-tutorials-03-shaders-viz-shader-py>`_ shaders example as a starting point and just replaced the `utah` actor by Rectangle2D actor. This program executed successfully without any errors, but it didn't give the required output.
+
+So instead of wasting time searching for how it is done, we thought it would be great if we directly ask in vtk forum. For this, I had to create a minimalistic pure vtk code. You can check out my code as well as the post `here <https://discourse.vtk.org/t/how-to-connect-shaders-to-actor2d/9384>`_.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:29]: Maybe `in the VTK discourse forum` is more clear

Comment (m-agour) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:29]: Also `vtk` should be  `VTK`

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+.. post:: September 14 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I started with highlighting the shapes. As discussed earlier I had two ways but while implementing them, I found out both ways aren't that good to continue with.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:13]: `As discussed earlier, I had two ways, but while...`

**Review by m-agour — COMMENTED** [review]
Hello @ganimtron-10,
Great post you wrote. Please take a look at my comments below, thanks.

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst (hunk 2 above)
Comment (xtanion) [reply docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:29]: ++

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+The first way in which we thought of creating the scaled shapes in the background had an issue with the stacking. The border(blue rectangle) and the shape(grey rectangle) both seem to look like different shapes just grouped together as shown below.
+
+.. image:: https://user-images.githubusercontent.com/64432063/192321622-964cef6e-f965-4a24-8dcf-0b899fe5e387.gif
+    :width: 400
+    :align: center
+
+While playing around with the second way, which was to add yellow color to the shape to make it brighter, it was difficult to identify which shape was selected also sometimes instead of making it brighter the addition of color created a new color which again confused the user.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:20]: `;` or `.` after `selected`

File: docs/source/posts/2022/2022-09-14-week-15-praneeth.rst (hunk 3 above)
Comment (xtanion) [inline docs/source/posts/2022/2022-09-14-week-15-praneeth.rst:13]: comma after `earlier`

**Review by xtanion — COMMENTED** [review]
Hi @ganimtron-10, Overall the blog LGTM. Take a look at Mohamed's comment, He has covered most of the punctuation mistakes.

**Review by skoudoro — COMMENTED** [review]
Hi @ganimtron-10,

See my comment below, then it is ready to go. thanks @ganimtron-10

**Comment by ganimtron-10** [comment]
Hello @skoudoro ,
I updated the blog as per the review.

**Comment by skoudoro** [comment]
thank you @ganimtron-10, merging

**Pull request merged**